pkg/compilers/osv/osv_dynamic.go

```diff
@@ -0,0 +1,82 @@
+package osv
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/emc-advanced-dev/pkg/errors"
+	unikos "github.com/emc-advanced-dev/unik/pkg/os"
+	"github.com/emc-advanced-dev/unik/pkg/types"
+	unikutil "github.com/emc-advanced-dev/unik/pkg/util"
+	"gopkg.in/yaml.v2"
+)
+
+const OSV_DEFAULT_SIZE = "10GB"
```

Comment:
Is there any reason we shouldn't reduce this to 1GB? The only way I could ever go over 100MB was when I added some data to the unikernel.